src/app/validators/custom-validators.ts

```diff
@@ -195,6 +195,35 @@ export class CustomValidators {
     };
   }
 
+  // Set this on both password and confirmation fields so it runs when either changes
+  // confirm should be false for the confirmation field validator
+  static unmatchPassword(unmatch: string, confirm: boolean): ValidatorFn {
```

Comment:
This function is a near copy of `matchPassword`.  Please try to refactor to use `matchPassword` for both so we don't have duplication here.